php-checks/src/main/java/org/sonar/php/checks/UnusedFunctionParametersCheck.java

```diff
@@ -63,8 +63,14 @@ public void visitMethodDeclaration(MethodDeclarationTree tree) {
   }
 
   private void checkParameters(FunctionTree tree) {
+    boolean hasFuncGetArgs = false;
+    if (tree instanceof FunctionDeclarationTree)
+      hasFuncGetArgs = Symbols.get((FunctionDeclarationTree) tree).hasFuncGetArgs();
+    if (tree instanceof MethodDeclarationTree)
+      hasFuncGetArgs = Symbols.get((MethodDeclarationTree) tree).hasFuncGetArgs();
+
     Scope scope = context().symbolTable().getScopeFor(tree);
-    if (scope != null && !scope.hasUnresolvedCompact()) {
+    if (scope != null && !scope.hasUnresolvedCompact() && !hasFuncGetArgs) {
```

Comment:
```suggestion
    if (!(scope == null || hasFuncGetArgs || scope.hasUnresolvedCompact())) {
```
Performance-wise it's better like this